<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GL::Internal::BehaviorSettingsPage</class>
 <widget class="QWidget" name="GL::Internal::BehaviorSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>343</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>User Interface</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item row="4" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="zoomSpeedLabel">
          <property name="text">
           <string>Zoom Speed:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>285</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="zoomSpeedSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Viewports</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item row="4" column="0">
       <layout class="QGridLayout" name="gridLayout_2"/>
      </item>
      <item row="4" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>285</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="linkModelTranslationsCheckBox">
        <property name="text">
         <string>Link Model Translations</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
